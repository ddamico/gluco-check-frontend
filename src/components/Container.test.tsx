import React from "react";
import { cleanup, render } from "@testing-library/react";
import { axe, toHaveNoViolations } from "jest-axe";
import Container from "./Container";

expect.extend(toHaveNoViolations);

afterEach(cleanup);

describe("Container component", () => {
  it("renders the component", () => {
    const { container } = render(<Container>test</Container>);
    expect(container.firstChild).toMatchSnapshot();
  });

  it("has no axe violations", async () => {
    const { container } = render(<Container>test</Container>);
    expect(await axe(container)).toHaveNoViolations();
  });
});
