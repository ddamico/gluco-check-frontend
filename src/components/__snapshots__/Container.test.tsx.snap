// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component renders the component 1`] = `
<div>
  test
</div>
`;
